config/Dockerfiles/tests.d/aws/05_aws_ec2_vpc_endpoint

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
 # Verify the aws ec2 vpc subnet provisioning
-# distros.exclude: centos7 fedora27 fedora28 fedora30
+# distros.exclude: centos7 fedora29 fedora30
```

Comment:
We should still be running this on at least one distro

Reply:
Will do.